### PR TITLE
Add document.titles to pages

### LINKED ISSUE #174: Update page titles
When you navigate to a new page, the page title should be updated depending on what is on that page.

### PR DESCRIPTION
**Pull request by p-netm**
closes #174

### CODE REVIEW

**Comment by p-netm** [comment]
Views without a custom titile:
- oath pages login, logout, outh pages
- untested title for IRS pages
This has been pushed to another issue #203

**Comment by p-netm** [comment]
pageTitles

File: src/containers/pages/FocusInvestigation/map/active/index.tsx [hunk 1]
```diff
@@ -197,6 +200,9 @@ class SingleActiveFIMap extends React.Component<
 
     return (
       <div>
+        <Helmet>
+          <title>{FI_SINGLE_MAP_TITLE}</title>
```
Comment (moshthepitt) [inline src/containers/pages/FocusInvestigation/map/active/index.tsx]: This title should include the title of the plan, something like: `Focus Investigation: A1 - Tha Sen 8- Focus 01 |  Reveal`

File: src/containers/pages/FocusInvestigation/single/index.tsx [hunk 2]
```diff
@@ -169,6 +171,9 @@ class SingleFI extends React.Component<RouteComponentProps<RouteParams> & Single
 
     return (
       <div className="mb-5">
+        <Helmet>
+          <title>{FI_SINGLE_TITLE}</title>
```
Comment (moshthepitt) [inline src/containers/pages/FocusInvestigation/single/index.tsx]: This title should be the name of the focus area, something like: `Focus Investigations in Tha Sen 8 |  Reveal`

File: src/constants.tsx [hunk 3]
```diff
@@ -98,6 +98,16 @@ export type FI_SINGLE_MAP_URL = typeof FI_SINGLE_MAP_URL;
...
 export type FI_HISTORICAL_URL = typeof FI_HISTORICAL_URL;
 
+// container pages title names
+export const HOME_TITLE = `Home page`;
+export type HOME_TITLE = typeof HOME_TITLE;
+export const FI_ACTIVE_TITLE = `Active Focus Investigation`;
+export type FI_ACTIVE_TITLE = typeof FI_ACTIVE_TITLE;
+export const FI_HISTORICAL_TITLE = `Historical Focus Investigations`;
+export type FI_HISTORICAL_TITLE = typeof FI_HISTORICAL_TITLE;
+export const IRS_TITLE = `Indoor residual Spray`;
```
Comment (moshthepitt) [inline src/constants.tsx]: Please lets have this as `IRS`

**Pull request merged**